### PR TITLE
Carousel single spot

### PR DESCRIPTION
**Pull request by eamurao**
1. Customize No. of videos to show : SUCCESS
2. Show carousel in "/test-carousel-spotlight/iframe-static/": DONE
3. Customzie style: border(color, thickness, style) / header(background color, font color, text alignment): done
4. Fix modal products and hover

### CODE REVIEW

File: static/carousel/js/index.js [hunk 1]
```diff
@@ -111,12 +126,14 @@
 
                     carouselSettings.onClick = function (clicked,videos) {
                         if (window.parent) {
-                            window.parent.postMessage({
+                            window.parent.postMessage({
```
Comment (rfornes) [inline static/carousel/js/index.js]: what's the change here?

File: static/carousel/js/index.js [hunk 2]
```diff
@@ -111,12 +126,14 @@
...
                         if (window.parent) {
-                            window.parent.postMessage({
+                            window.parent.postMessage({                                
                                 runTime: 'undefined' !== typeof window.__TVPage__ ? __TVPage__ : null,
                                 event: 'tvp_carousel:video_click',
                                 selectedVideo: clicked,
                                 videos: videos
                             }, '*');
+                            
+                            console.log(window.parent);
```
Comment (rfornes) [inline static/carousel/js/index.js]: ?

File: static/carousel/js/carousel.js [hunk 3]
```diff
@@ -83,7 +121,7 @@
                 });
 
                 $carousel.slick({
-                    slidesToShow: 3,
+                    slidesToShow: config.no_of_videos,
```
Comment (rfornes) [inline static/carousel/js/carousel.js]: Did you declared `no_of_videos ` and the other options on `defaults/carousel.json` ?

File: static/carousel/js/index.js (hunk 1 above)
Comment (eamurao) [inline static/carousel/js/index.js]: only space (whitespace)

File: static/carousel/js/index.js (hunk 2 above)
Comment (eamurao) [inline static/carousel/js/index.js]: I forgot to remove this. It's for my testing

File: static/carousel/js/carousel.js (hunk 3 above)
Comment (eamurao) [inline static/carousel/js/carousel.js]: No. From root/content/tvpwidget/carouse-1.md

Comment (rfornes) [inline static/carousel/js/carousel.js]: Ok, please declare this on  `defaults/carousel.json` as well. you will see other options in that  file that will help you as example

File: static/carousel/js/index.js (hunk 1 above)
Comment (rfornes) [inline static/carousel/js/index.js]: cool

File: static/carousel/js/carousel.js [hunk 4]
```diff
@@ -61,6 +97,8 @@
 
             this.container.appendChild(carouselFrag);
 
+            var config = getSettings('static');
+            console.log(config);
```
Comment (rfornes) [inline static/carousel/js/carousel.js]: ?

File: static/carousel/js/index.js [hunk 5]
```diff
@@ -143,7 +158,7 @@
                         }, '*');
                     }
                 };
-
+                // customize(carouselSettings);
```
Comment (rfornes) [inline static/carousel/js/index.js]: ?

**Pull request closed without merging**